packages/flutter/lib/src/material/stepper.dart

```diff
@@ -717,7 +728,7 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
               AnimatedSize(
                 curve: Curves.fastOutSlowIn,
                 duration: kThemeAnimationDuration,
-                child: widget.steps[widget.currentStep].content,
+                child: Column(children: stepPanels, crossAxisAlignment: CrossAxisAlignment.stretch),
```

Comment:
Would like a second eye on this `crossAxisAlignment` value, which I _think_ is what best replicates the old layout; but I am not certain.

Reply:
I think this looks ok.